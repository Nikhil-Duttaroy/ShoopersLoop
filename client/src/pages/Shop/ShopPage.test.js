import React from "react";
import { shallow } from "enzyme";
import { ShopPage } from './ShopPage.components';


describe('ShopPage', () => {
  let mockMatch;

  beforeEach(() => {
    mockMatch = {
      path: "/shop",
    };
  });

  it("should render ShopPage component", () => {
    expect(shallow(<ShopPage match={mockMatch.path} />)).toMatchSnapshot();
  });
});


