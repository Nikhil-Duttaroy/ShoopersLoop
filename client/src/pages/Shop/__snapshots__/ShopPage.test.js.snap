// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShopPage should render ShopPage component 1`] = `
<Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
    prioritizeSeoTags={false}
  >
    <title>
      ShoopersLoop: Shop
    </title>
    <meta
      content="Shoppage for shoopersloop , choose from your favorite clothing categories"
      name="description"
    />
    <link
      href="/shop"
      rel="canonical"
    />
  </Helmet>
  <div
    className="shop-page"
  >
    <Suspense
      fallback={<Spinner />}
    >
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_init": [Function],
            "_payload": Object {
              "_result": [Function],
              "_status": -1,
            },
          }
        }
        exact={true}
        path="undefined"
      />
      <Route
        component={
          Object {
            "$$typeof": Symbol(react.lazy),
            "_init": [Function],
            "_payload": Object {
              "_result": [Function],
              "_status": -1,
            },
          }
        }
        exact={true}
        path="undefined/:collectionId"
      />
    </Suspense>
  </div>
</Fragment>
`;
